tests/test_select_by_mask.py

```
@@ -0,0 +1,144 @@
+        obj.select_by_mask(np.array([0, 1, 1]))
+
+
+def test_select_by_mask_rejects_length_mismatch(obj):
+    with pytest.raises(ValueError, match="does not match first dimension"):
+        obj.select_by_mask(np.array([True, False]))
+
+
+class TestLazyMaskIsCopied:
+
+    def test_lazy_arraydict(self, test_filepath):
+        data, f = _make_lazy(_make_array_dict(), LazyArrayDict, test_filepath)
```

Ruff (0.15.13)

[warning] Unpacked variable `f` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)